src/VersionControl/GitVersionControl.php

```diff
@@ -42,7 +42,7 @@ public function getStagedFiles()
         foreach ($this->getFiles() as $file) {
             list($status, $relativePath) = explode("\t", $file);
 
-            $fullPath = $base . DIRECTORY_SEPARATOR . $relativePath;
+            $fullPath = trim($base . DIRECTORY_SEPARATOR . $relativePath);
```

Comment:
Feel free to open a new PR with the change. I saw no reason for this to hold up a merge though.

It'd be helpful to explain why `rtrim` would be better than `trim`.